editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -237,20 +247,32 @@ function parseObjectLiteralExpression(
   let highlightBounds = existingHighlightBounds
   let propsUsed: Array<string> = []
   let definedElsewhere: Array<string> = []
+
+  // Get comments attached to the open brace.
+  let openBraceComments: ParsedComments = emptyComments
+  const firstChild = literal.getChildAt(0, sourceFile)
+  if (firstChild != null && firstChild.kind === TS.SyntaxKind.OpenBraceToken) {
```

Comment:
isn't this the same code as `getJSXExpressionLeadingComments` ?

Reply:
Very similar, I'll likely be revisiting this when finishing off the work around including comments in JSX attributes.